implementation.md

```diff
@@ -419,8 +429,28 @@ is used to discover the properties of the action.
 
 For details on the parameters to an action, see below.
 
+These two methods are a bit different in how they operate. Actions provided by
+a method are limited by the instances, or one globally. For instance, the SFTP
+plugin allows creating file deletion commands. There's only one way to delete a
+file, so it is in a method per instance. The `Definer` method allows
+dynamically creating actions. For instance, the Vidarr plugin will scan the
```

Comment:
_The_ Vidarr plugin? Is this meant to say server or is this a particular plugin?